packages/sample-app/src/components/common/ErrorBoundary.tsx

```diff
@@ -7,7 +7,7 @@ export type ErrorBoundaryFallbackProps = {
   errorInfo: React.ErrorInfo;
 };
 
-type ErrorBoundaryProps = AnyObject;
+type ErrorBoundaryProps = React.PropsWithChildren<AnyObject>;
```

Comment:
FWIW there is a ticket in OCP console to expose our common error boundary fallback: https://issues.redhat.com/browse/CONSOLE-3276. Maybe we could eventually use a common one here in lieu of a custom sample one.